.github/opensearch-cluster/docker-compose.yml

```diff
@@ -8,6 +8,7 @@ services:
       - '9600:9600'
     environment:
       - 'OPENSEARCH_INITIAL_ADMIN_PASSWORD=${OPENSEARCH_PASSWORD:-myStrongPassword123!}'
+      - 'OPENSEARCH_JAVA_OPTS=${OPENSEARCH_JAVA_OPTS}'
```

Comment:
In my experiments these quotes are unnecessary on both `OPENSEARCH_JAVA_OPTS` and `OPENSEARCH_INITIAL_ADMIN_PASSWORD`, so you can remove them.